package.json

```diff
@@ -94,13 +100,5 @@
     "testEnvironmentOptions": {
       "enzymeAdapter": "react16"
     }
-  },
-  "husky": {
-    "hooks": {
-      "pre-commit": "lint-staged"
-    }
-  },
-  "lint-staged": {
-    "*.{js,jsx,ts,tsx}": "npm run lint:fix"
```

Comment:
I can't work with this... is this fine for you? I think if you want you can add these on commit checks for yourself

Reply:
I'm fine with removing.